packages/ra-ui-materialui/src/list/SimpleList/SimpleList.tsx

```diff
@@ -121,6 +122,7 @@ export const SimpleList = <RecordType extends RaRecord = any>(
                         rowClick={rowClick}
                         rowSx={rowSx}
                         rowStyle={rowStyle}
+                        resource={resource}
```

Comment:
This fixes an issue we somehow missed before: `SimpleList` was not propagating its resource prop